pkg/blackduck/ctl_blackduck.go

```diff
@@ -340,8 +341,14 @@ func (ctl *Ctl) SetFlag(f *pflag.Flag) {
 			ctl.Spec.AuthCustomCA = data
 		case "type":
 			ctl.Spec.Type = ctl.Type
-		case "desired-state":
-			ctl.Spec.DesiredState = ctl.DesiredState
+		case "start":
```

Comment:
what will happen if both are set? :-)

Reply:
Maybe there should be only one flag?  What does it mean to have `stop` = false?